assets/app/view/game/part/town_rect.rb

```diff
@@ -144,27 +196,43 @@ def rotation_angles
                     end
 
             [(edge_a * 60) + delta]
+          elsif edge_a
+            [edge_a * 60]
+          else
+            # This town is in the center. Find the orientation of the town
+            # by looking at the first path connected to it
+            path = town.paths.first
+            if !path.edges.empty?
```

Comment:
if path.edges.any?

Reply:
Sure. Interesting that rubocop missed that.